packages/core-modules/test/contracts/modules/ElectionModule.schedule.test.js

```diff
@@ -207,41 +207,24 @@ describe('ElectionModule (schedule)', () => {
       });
 
       it('shows that the current period is Nomination', async function () {
-        assertBn.eq(await ElectionModule.getCurrentPeriod(), ElectionPeriod.Nomination);
+        assertBn.eq(await ElectionModule.getCurrentPeriodType(), ElectionPeriod.Nomination);
       });
 
       itAcceptsNominations();
       itRejectsVotes();
       itRejectsEvaluations();
       itRejectsAdjustments();
-
-      describe('when fast forwarding within the current period', function () {
```

Comment:
why removing those usecases?

Reply:
I thought it was redundant and made tests harder to read